src/main/kotlin/org/rust/ide/annotator/fixes/ConvertToTyFix.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.annotator.fixes
+
+import com.intellij.codeInspection.LocalQuickFixAndIntentionActionOnPsiElement
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsExpr
+import org.rust.lang.core.psi.RsPsiFactory
+import org.rust.lang.core.types.ty.Ty
+
+/**
+ * For the given `expr` converts it to the type `ty` with `ty::from(expr)`
+ */
+class ConvertToTyFix(expr: PsiElement, val ty: Ty) : LocalQuickFixAndIntentionActionOnPsiElement(expr) {
+    override fun getFamilyName(): String = "Convert to type"
+
+    override fun getText(): String = "Convert to type $ty"
+
+    override fun invoke(project: Project, file: PsiFile, editor: Editor?, startElement: PsiElement, endElement: PsiElement) {
+        if (startElement !is RsExpr) return
+        startElement.replace(RsPsiFactory(project).createStaticMethodCall(ty.toString(), "from", listOf(startElement)))
```

Comment:
We can't pass `ty.toString()` as type name here because if `ty` is  generic it uses wrong syntax
For instance, 
```rust
let v: Vec<u8> = String::new();
```
should be converted into 
```rust
let v: Vec<u8> = Vec::<u8>::from(String::new());
```
or even into
```rust
let v: Vec<u8> = Vec::from(String::new());
```
but current implementation tries to use 
```rust
let v: Vec<u8> = Vec<u8>::from(String::new());
```
syntax and fails

Reply:
And let's add tests for such cases

Reply:
Yeah i missed that. 

Given that we have the expected type, from the top of my head the ` Vec::from(String::new())` version seem to  work everywhere where `Vec::<u8>::from(String::new())` works. So, i guess, it is safe to always skip the type arguments in this case, does that sound right?

Reply:
Yeah, sounds reasonable

Reply:
done (maybe, not sure if that's the best way to do that)